src/CssInliner.php

```diff
@@ -179,7 +179,7 @@ public function inlineCss(string $css = ''): self
         if ($this->isStyleBlocksParsingEnabled) {
             $combinedCss .= $this->getCssFromAllStyleNodes();
         }
-        $parsedCss = new CssDocument($combinedCss);
+        $parsedCss = new Document($combinedCss);
```

Comment:
I think it should perhaps be `use`d `as` `CssDocument`.  Reading through the code makes one wonder: "what sort of document?"